### ISSUE TITLE
TextArea – the label is read out together with the span information, even though they are not connected

### ISSUE THREAD

**Issue by havvamehfooz** [FreedomScientific/VFO-standards-support#906]
Summary

When span is placed in the html content, followed by a label and certain css is applied, in JAWS VPC mode, when the user reaches the span via arrow keys, the label is read out together with the span information, even though they are not connected.

Plain HTMl example:
https://jsbin.com/jigiquxeci/edit?html,output

Steps to reproduce:
When CSS is added with link tag, the issue is reproducible. When css is commented or remove the class="sapMLabelInner" is removed, the issue is not reproducible.

**Comment by JeremiahRogersVispero** [comment]
Can you provide additional reproduction guidance or samples please, @havvamehfooz ? I cannot reproduce the issue using JAWS 2025.2505.43 and Chrome 136. Thanks in advance.
